corehq/apps/domain/models.py

```diff
@@ -421,6 +423,10 @@ def wrap(cls, data):
         if 'location_types' in data:
             data['obsolete_location_types'] = data.pop('location_types')
 
+        if 'granted_messaging_access' not in data:
+            # enable messaging for domains created before this flag was added
+            data['granted_messaging_access'] = True
```

Comment:
@gcapalbo: I discussed this approach with @dannyroberts and also reviewed places where `Domain.wrap(data)` is called throughout the codebase. This seems to be the recommended and easiest way to add a property to an existing couch model with a value that is different from the value to be applied to existing documents. Another alternative would be to make a migration to run against existing data, but that has its own special pain.